Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -56,20 +51,20 @@ require`. The full syntax is::
 
 **Example**::
 
-    composer require typo3/cms:~7.6.0
+    composer require typo3/cms:~9.5.0
```

Comment:
This is impossible, here we must also use `typo3/minimal`. Also: why not `^9.5` or `^9`?

Reply:
done